microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/definition/AbstractAnnotationDefinitionParticipant.java

```diff
@@ -133,11 +130,7 @@ public boolean isAdaptedForDefinition(JavaDefinitionContext context, IProgressMo
 		}
 
 		// Get the annotation member value range
-		ISourceRange r = ((ISourceReference) annotation).getSourceRange();
-		int offset = annotationSource.indexOf(annotationMemberValue);
-		IJDTUtils utils = context.getUtils();
-		final Range annotationMemberValueRange = utils.toRange(typeRoot, r.getOffset() + offset,
-				annotationMemberValue.length());
+		final Range annotationMemberValueRange = annotationMemberInfo.getRange();
 
 		if (definitionPosition.equals(annotationMemberValueRange.getEnd())

```

Comment:
This code can be removed since this test isdone in getMemberValueAt, no?

Reply:
Which test are you referring to here?

Reply:
I mean https://github.com/eclipse/lsp4mp/pull/216/files#diff-0e83b064456d135fec98f39279337fdc598349adf9ccd8bc92c992e228d336d7R267